docs/api/monitoring/monitoring.md

```diff
@@ -3,23 +3,32 @@ id: monitoring
 title: Monitoring Interface
 ---
 
-A monitoring object's interface doubles the interface of a monitoring adapter except for the [`getMetrics`](#getmetrics) and [`clearMetrics`](#clearmetrics) methods that require a monitoring adapted ID as an argument.
+:::info TypeScript Support
+
+A monitoring object has an associated TypeScript type:
+
+- Type Name - `Monitoring`
+- Package - `@resolve-js/core`
+
+:::
+
+A monitoring object's interface is the same as the interface of a monitoring adapter except for the [`getMetrics`](#getmetrics) and [`clearMetrics`](#clearmetrics) methods that require a monitoring adapted ID as an argument.
```

Comment:
Thanks for updating the sentence. I noticed one more thing.
Is "a monitoring adapted ID" correct, or do you mean "a monitoring adaptor ID"?

Reply:
Thanks, I fixed the typo

Reply:
Thank you.